### PR TITLE
Closed #139 - duplicate popin on user creation/edition

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Now we can't have multiple form user creation or edition.

this closes https://github.com/backbee/backbee-standard/issues/139

### CODE REVIEW

File: src/tb/apps/user/controllers/group.controller.js [hunk 1]
```diff
@@ -161,4 +161,4 @@ define(
             }
         });
     }
-);
\ No newline at end of file
+);
```
Comment (ndufreche) [inline src/tb/apps/user/controllers/group.controller.js]: you change nothing, can remove this file from the PR please ?

File: src/tb/apps/page/views/page.view.tree.js [hunk 2]
```diff
@@ -297,4 +296,4 @@ define(
 
         return PageViewTree;
     }
-);
\ No newline at end of file
+);
```
Comment (ndufreche) [inline src/tb/apps/page/views/page.view.tree.js]: You change nothing, can remove this file from the PR please ?

File: src/tb/component/formbuilder/form/Form.js [hunk 3]
```diff
@@ -96,7 +96,6 @@ define(
              * @param {Object} config
              */
             computeDefaultValue: function (config) {
-
```
Comment (ndufreche) [inline src/tb/component/formbuilder/form/Form.js]: You change nothing, can remove this file from the PR please ?

**Comment by ndufreche** [comment]
:+1: Good job.

**Comment by mickaelandrieu** [comment]
@ndufreche comments adressed.

**Comment by ndufreche** [comment]
great thanks @mickaelandrieu

**Pull request merged**